modules/acquia_cms_headless/tests/src/Functional/DashboardApiDocumentTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+use Drupal\Tests\acquia_cms_headless\Traits\DashboardTableTrait;
+
+/**
+ * Tests headless dashboard API Document.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class DashboardApiDocumentTest extends DashboardWebDriverTestBase {
+
+  use DashboardTableTrait;
```

Comment:
@deepakmishra2 Please remove this trait if your test include testing with table. Same for below test as well

Reply:
+1

Reply:
Done